runtime/src/main/jni/ManualInstrumentation.cpp

```diff
@@ -0,0 +1,8 @@
+//
+// Created by Panayot Cankov on 26/05/2017.
+//
+
+#include "ManualInstrumentation.h"
+
+bool tns::instrumentation::Frame::disabled = true;
+const std::chrono::steady_clock::time_point tns::instrumentation::Frame::disabled_time = std::chrono::steady_clock::now();
```

Comment:
std::chrono::steady_clock::time_point(); - the empty constructor defaults to duration::zero which should be good enough for a "disabled" value.